docs/src/documentation/03-components/08-visuals/seat/01-guidelines.mdx

```diff
@@ -1,3 +1,57 @@
 ---
 title: Guidelines
 ---
+
+<ReactExample name="Seat" />
+
+## When to use
+
+- To show information about options for where to sit.
+- To enable users to scan the distinct types to make a selection.
+
+### When not to use
+
+- For general visual context---use an [illustration](/components/visuals/illustration/).
+
+## Component status
+
+<ComponentStatus component="Seat" />
+
+## Content structure
+
+<FigmaFile
```

Comment:
seems like you forgot to import `ComponentStatus` and `FigmaFile` 🤔  Or you plan to add it later?

Reply:
It should be a global added with https://github.com/kiwicom/orbit/pull/2658